### DISCUSSION TITLE
Walkups (Sweeper or Voice Track Over Song Intro)

### DISCUSSION THREAD

**Discussion by wppronto** [AzuraCast/AzuraCast#6941] [Q&A]
I'd love to automate things so that sweepers and/or voice tracks can do proper walkups (voice ends and the singing begins). I know this isn't part of AzuraCast, and it would require multiple things including an intro cue point. I could swear somewhere I saw someone's Liquidsoap configuration that accomplished this.

I'm wondering if there are any plans to add anything like this, possibly including general support for voice tracking? Or can anyone point me toward that Liquidsoap configuration if it really exists?

I'm happy to try learning things and sharing my results, but I'm a novice with AzuraCast and Liquidsoap at this point.

Thanks!

**Comment by t0mtaylor** [comment]
Voice tracking would be great - is this on the roadmap @BusterNeece ?

**Reply by DJMic** [reply]
I think it depends on liquidsoap. I had asked this question on their Github but it seems it wasn't planned when I asked this question last year.

**Reply by t0mtaylor** [reply]
Its not liquid soap thats the issue, this is a massive project - to store voice notes, then the UI shows how it blends into position with the log output

**Comment by t0mtaylor** [comment]
@BusterNeece @wppronto @DJMic I started work on Voice Tracking, im upto the point where i need to generate the logs ahead of time to voice tracking can be inserted
![Presenter VT](https://github.com/user-attachments/assets/46259512-6b92-47b7-a1d8-1cc8c72a861d)
![Admin VT Plan](https://github.com/user-attachments/assets/98c15f59-9a2b-4794-8790-5302a4cc4796)

**Reply by wppronto** [reply]
That is fantastic news! I've been away but getting back into it. While the next week is busy, after that I'm happy to help test!

**Reply by t0mtaylor** [reply]
@wppronto May be a while this is a massive job, i'll put a video together once its working

**Reply by DJMic** [reply]
Great job, that's fantastic! I'd like to test this new feature. What will you use to generate the logs ? Will it be based on rotations between differents playlists with their respective weights ?

**Reply by t0mtaylor** [reply]
@DJMic thats right this is just meant to slot into place and use the existing log builder, then in the voice tracking hour you can set limits on where links can go between the tracks - you can also drag and re-order the log items (but they are locked when a VT link is set) - there will be station wide overrides too and rules you can set - but i'll show some more screenshots and a video once thats all working

also going to reuse the media and waveform thats in the media but it will be enhanced for the out track, vt link and in track with segue options, etc
